metaflow/package.py

```diff
@@ -74,6 +76,20 @@ def path_tuples(self):
                     exclude_hidden=False,
                     addl_suffixes=self.metaflow_custom_addl_suffixes):
                 yield path_tuple
+             
+        # Any custom packages exposed via decorators
+        deco_module_paths = set()
+        for step in self._flow:
+            for deco in step.decorators:
+                for path_tuple in deco.add_to_package():
+                    file_path,_ = path_tuple
```

Comment:
Out of curiosity, what is the second element of the tuple?

Reply:
`add_to_package` returns a list of (path, arcname) to be added to the job package, where `arcname` is the alternative name for the file in the package. This follows the same convention as `MetaflowPackage.path_tuples`. 

An example implementation of this decorator hook can be seen [here](https://github.com/valayDave/metaflow/blob/c6d9260ec32c7bdc13a8fb1f728197f5af135133/metaflow/plugins/cards/card_decorator.py#L26).